src/deserialize_sequence.cc

```diff
@@ -93,7 +94,7 @@ void extract_type(gul14::string_view extract, Step& step)
         case "end"_sh:
             step.set_type(Step::type_end); break;
         default:
-            throw Error(gul14::cat("type: unable to parse ('", keyword, "')"));
+            throw Error(gul14::cat("type: unable to parse (\"", escape(keyword), "\")"));
```

Comment:
Don't we need then also an `unescape` in `serialize_sequence.cc`?

Reply:
The escape just happens when the user-readable string (error message) is created.  
This is not in the (de)serialization.